### PR TITLE
Document the process for how to stop sending

### PR DESCRIPTION
**Pull request by jasonpaige**
During live operation there will be points in time where it'll be
desirable to prevent the sending of all data for printing and posting.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: doc/arch/adr-001-stop-app-to-stop-sending.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+On the face of it, this sounds like a feature switch, and this is likely to be the case strategically. Given the short timescales for go live for this product, this may not be achievable.
+
+## Decision
+To fulfil this requirement, the send-letter-consumer-service will simply be disabled from Azure Portal.
+
+## Status 
+Accepted
+
+## Consequences
+This menas involving those with higher permissions withing Azure Portal in order to disable an application. This shouldn't be an issues as these are typically the people involved with the day to day operations of the systems.
```
Comment (nitinprabhu) [inline doc/arch/adr-001-stop-app-to-stop-sending.md]: typo error `means`

Comment (rkondratowicz) [inline doc/arch/adr-001-stop-app-to-stop-sending.md]: typo in 'means'

File: doc/arch/adr-001-stop-app-to-stop-sending.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# ARD 001: Stop app to stop sending
```
Comment (rkondratowicz) [inline doc/arch/adr-001-stop-app-to-stop-sending.md]: file name uses 'adr', this line uses 'ard', which one is correct?

**Review by rkondratowicz — CHANGES_REQUESTED** [review]
typos

File: doc/arch/adr-001-stop-app-to-stop-sending.md (hunk 2 above)
Comment (jasonpaige) [reply doc/arch/adr-001-stop-app-to-stop-sending.md]: adr ;) architecture decision register

**Review by rkondratowicz — APPROVED** [review]

**Pull request merged**